src/bigquery.ts

```diff
@@ -1690,6 +1712,15 @@ export class BigQuery extends common.Service {
   getJobs(options?: GetJobsOptions): Promise<GetJobsResponse>;
   getJobs(options: GetJobsOptions, callback: GetJobsCallback): void;
   getJobs(callback: GetJobsCallback): void;
+  /**
+   * @callback GetJobsCallback
+   * @param {?Error} err Request error, if any.
+   * @param {object} jobs An array of [Job]{@link https://cloud.google.com/bigquery/docs/reference/v2/jobs} objects.
```

Comment:
Is this another node-ism, that the callback args apparently have key names but the response objects use integers for argument ordering?

Reply:
Yeah, that's correct. It difference looks like this:
```
bigquery.getJobs(function(err, jobs) {
 if (!err) {
   // jobs is an array of Job objects.
  }
});

bigquery.getJobs().then(function(data) {
   const jobs = data[0];
});
```